### PR TITLE
fix: cleanse inputs that may have issues

### LINKED ISSUE #124: Spaces in the text attribute throw
postgres thinks this is a string search and the quote is unterminated. I wonder if all quotes should be stripped. Double quotes would be considered a column etc so I don't think anything good will come from it. 

e.g. `/suggest?f=json&text=3000 N MAIN ST, LOGAN'&maxSuggestions=6`

### PR DESCRIPTION
**Pull request by stdavis**
Closes #124

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 setup(
     name='masquerade',
-    version='1.1.7',
```
Comment (steveoh) [inline setup.py:14]: **question**: do you want the version bump in here?

**Review by steveoh — APPROVED** [review]
i wonder if we'd want any of the other [api client type of replacements](https://github.com/agrc/api-client/blob/main/src/services/geocode.js#L15) added here?

File: setup.py (hunk 1 above)
Comment (stdavis) [reply setup.py]: Oops. Wrong target branch. I switched it to dev. And TBH - I can't remember if the version bump is automated or not. I'll check on it when I promote from dev to main.

**Comment by stdavis** [comment]
>i wonder if we'd want any of the other [api client type of replacements](https://github.com/agrc/api-client/blob/main/src/services/geocode.js?rgh-link-date=2023-06-05T22%3A23%3A17Z#L15) added here?

I feel like we looked at re-implementing those in this project and decided that it was best to keep them in one place. But I'm open to different thoughts on it for sure.

**Pull request merged**